tests/components/rfxtrx/test_config_flow.py

```diff
@@ -1103,6 +1103,87 @@ async def test_options_add_and_configure_device(hass):
     assert "delay_off" not in entry.data["devices"]["0913000022670e013970"]
 
 
+async def test_options_configure_rfy_cover_device(hass):
+    """Test we can configure the venetion blind mode of an Rfy cover."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            "host": None,
+            "port": None,
+            "device": "/dev/tty123",
+            "automatic_add": False,
+            "devices": {},
+        },
+        unique_id=DOMAIN,
+    )
+    entry.add_to_hass(hass)
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
```

Comment:
We need to set up the config entry before starting the options flow. Use `hass.config_entries.async_setup`.

Reply:
Done.